packages/flutter_android_lifecycle/example/ios/Runner/AppDelegate.h

```diff
@@ -0,0 +1,6 @@
+#import <Flutter/Flutter.h>
```

Comment:
Do we want to commit example app iOS code? Would it be worth ripping it out? It includes some binary files in our checkout that it would be nice to avoid adding.

Reply:
Yes, it's important to verify that iOS apps build with this plugin as a dependency.